specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/artifacts.json

```diff
@@ -203,6 +203,52 @@
         }
       }
     },
+    "/linkedservices/{linkedServiceName}/rename": {
+      "post": {
+        "tags": [
+          "linkedServices"
+        ],
+        "operationId": "LinkedService_RenameLinkedService",
+        "x-ms-examples": {
+          "LinkedServices_Rename": {
+            "$ref": "./examples/LinkedServices_Rename.json"
+          }
+        },
+        "description": "Renames a linked service.",
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
```

Comment:
I'm also wondering if this API should support an `If-Match` header parameter, which would provide for the ability to prevent a resource getting renamed, a new one being created and a separate attempt to rename the resource (which was intended as a rename for the original resource, not for the new replacement resource )

(I'm not terribly worried, but it might be a good idea)

Reply:
This is just a passthrough api . This is taken care by implementation APi.